src/components/CardItem/CardItem.tsx

```diff
@@ -0,0 +1,172 @@
+import styled from "styled-components";
+import "react-loading-skeleton/dist/skeleton.css";
+import Delorean from "../../shared/assets/images/delorean.jpg";
+import { BaseColor } from "../../config/color";
+import { Cars, User_Cars } from "../../shared/graphql/__generate__/generated";
+import { Link } from "react-router-dom";
```

Comment:
please configure your linter to put external libraries at the top of the file https://eslint.org/docs/latest/rules/sort-imports